ci/ios/create-vm/scripts/install-brew.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if which brew
+then
+    echo >&1 "brew is already installed, nothing to do here"
```

Comment:
Are all these `echo >&1 ...` necessary, since `stdout` is already the default? (=> `&1`)